### PR TITLE
[UPDATED] Regression data checked, add more parameter for p

### PR DESCRIPTION
**Pull request by puiyanng**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -29,17 +30,32 @@
 results = pd.DataFrame()
 results["true_y"] = yTest
 
-ada = md.fit_ada_boost(xTrain, yTrain)
-results["prediction"] = ada.predict(xTest)
-print("Adaboost Classifier")
+#
```
Comment (DanishAlsayed) [inline main.py]: Do not comment out functioning code

**Comment by DanishAlsayed** [comment]
Did you verify that the values in the regression data file are correct?

File: modelling.py [hunk 2]
```diff
@@ -52,12 +55,41 @@ def fit_random_forest(features, labels):
...
-    model = grid_search.fit(features, labels) 
+    svm = SVC(kernel='rbf', probability=True)
+    param_grid = {'C': [1e-3, 1e-2, 1e-1, 1, 10, 100, 1000], 'gamma': [0.001, 0.0001]}
+    grid_search = GridSearchCV(svm, param_grid, n_jobs = 8, verbose=1)
+    model = grid_search.fit(features, labels)
     return model
 
+def fit_KNN(features, labels, withTuning):
+    if withTuning:
+        param_grid = {'n_neighbors': list(range(1, 30)), 'p': [1, 2, 3, 4, 5]}
```
Comment (DanishAlsayed) [inline modelling.py]: what is `p` are only 5 values enough?

**Comment by puiyanng** [comment]
- Regression data checked
- add more parameter for p

**Pull request merged**